src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -119,19 +119,20 @@
   </resheader>
   <data name="FirstTimeWelcomeMessage" xml:space="preserve">
     <value>Welcome to .NET Core!
----------------------
-Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.
+----------------------------------
+Learn more about .NET Core: https://aka.ms/dotnet-docs
```

Comment:
I did it on purpose, because cut-and-paste from a line that grabs an end-of-line character (e.g., period) will break when pasted into a browser.